dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraMetaManager.java

```diff
@@ -87,7 +88,7 @@ public DoraMetaManager(AlluxioConfiguration conf,
     mUfsManager = ufsManager;
   }
 
-  private UnderFileSystem getUfsInstance(String ufsUriStr) {
+  public UnderFileSystem getUfsInstance(String ufsUriStr) {
```

Comment:
Why change this?

Reply:
just because you want to test this method?

Reply:
yes... for temporary

Reply:
OK, please don't modify the private access. The test of this method is no need.